WPMedia/news-theme-css
# Tmedia 383 deprecation notice math

if new to news-theme-css files, I created another pr as well to document files that may help https://github.com/WPMedia/news-theme-css/pull/76

## Description
Migrate from node-sass to dart-sass. And update deprecation notices, specifically math /

## Jira Ticket
- [TMEDIA-383](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-383)

## Acceptance Criteria
- No deprecation notices for running blocks with engine 3.0 

## Test Steps

1. `git checkout TMEDIA-383-deprecation-notice-math`
2. Remove previous deps and install new deps `rm -rf node_modules && npm i`
3. `npm run scss-watch` from news-theme-css folder to watch scss changes and prep for local dev
4. Open fusion-news-theme and include your path to news-theme-css in .env `CSS_FRAMEWORK_REPO=/{****your path***}/news-theme-css`
5. Use fusion version `FUSION_RELEASE=3.0.0`
5. LInk only news-theme-css `npx fusion start -f` 
6. You should not see any deprecation notices in fusion-verify 
7. Open http://localhost/pf/homepage/?_website=the-sun. Styles render

## Effect Of Changes
### Before

```
fusion-verify | Recommendation: math.div($number, $number * 0 + 1)
fusion-verify | 
fusion-verify | More info and automated migrator: https://sass-lang.com/d/slash-div
fusion-verify | 
fusion-verify | 72 │     @return $number / ($number * 0 + 1);

```

![Screen Shot 2021-07-30 at 14 11 48](https://user-images.githubusercontent.com/5950956/127701355-4495478c-f306-4624-8f22-688bd601e645.png)


### After

![Screen Shot 2021-07-30 at 14 11 16](https://user-images.githubusercontent.com/5950956/127701381-34d82ae0-72c2-4497-b808-e7b5bce126be.png)

## Dependencies or Side Effects

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
